src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/AnnotatedVariantProducer.java

```diff
@@ -323,4 +253,91 @@ private static VariantContext annotateWithImpreciseEvidenceLinks(final VariantCo
 
         return attributeMap;
     }
+
+    @VisibleForTesting
+    static VariantContextBuilder annotateWithExternalCNVCalls(final String recordContig, final int pos, final int end,
+                                                              final VariantContextBuilder inputBuilder,
+                                                              final Broadcast<SAMSequenceDictionary> broadcastSequenceDictionary,
+                                                              final Broadcast<SVIntervalTree<VariantContext>> broadcastCNVCalls,
+                                                              final String sampleId) {
+        if (broadcastCNVCalls == null || end < 0)
```

Comment:
Is this referring to `SvTYPE.NO_APPLICABLE_END`? If so please reference that. Also, what if we added methods `hasApplicableEnd` and `hasApplicableLength` to SvType and used those first instead of checking for negative numbers?